webui/react/src/components/LogViewer/LogViewer.tsx

```diff
@@ -4,6 +4,7 @@ import {
   ListChildComponentProps, ListOnItemsRenderedProps, ListOnScrollProps, VariableSizeList,
 } from 'react-window';
 import screenfull from 'screenfull';
+import Message, { MessageType } from 'shared/components/message';
```

Comment:
@stoksc I tried doing that before too. I tried again but it's still showing up...

Reply:
@stoksc that fixed it. But the UI checks are failing. I'm not sure why because rebased. And I don't see changes in the UI files? I ran make -C webui get-deps too.